app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UsersController < ApplicationController
+  include CurrentProject
+  include ProjectLevelAuthorization
+
+  helper_method :sort_column, :sort_direction
+
+  before_action do
+    find_project(params[:project_id])
+  end
+
+  before_action :authorize_project_admin!
+
+  def index
+    scope = User
+    scope = scope.search(params[:search]) if params[:search]
+    @users = scope.order(sort_column + ' ' + sort_direction).page(params[:page])
```

Comment:
Ups, my bad. I've fixed this in some places but missed this one. Thanks for the heads up.